pkg/kubelet/pod_workers.go

```diff
@@ -63,22 +72,28 @@ type workUpdate struct {
 
 func newPodWorkers(dockerCache dockertools.DockerCache, syncPodFn syncPodFnType, recorder record.EventRecorder) *podWorkers {
 	return &podWorkers{
-		podUpdates:  map[types.UID]chan workUpdate{},
-		isWorking:   map[types.UID]bool{},
-		dockerCache: dockerCache,
-		syncPodFn:   syncPodFn,
-		recorder:    recorder,
+		podUpdates:       map[types.UID]chan workUpdate{},
+		isWorking:        map[types.UID]bool{},
+		desiredPodsCache: map[types.UID]*api.BoundPod{},
+		dockerCache:      dockerCache,
+		syncPodFn:        syncPodFn,
+		recorder:         recorder,
 	}
 }
 
 func (p *podWorkers) managePodLoop(podUpdates <-chan workUpdate) {
+	var minDockerCacheTime time.Time
 	for newWork := range podUpdates {
 		// Since we use docker cache, getting current state shouldn't cause
-		// performance overhead on Docker. Moreover, as long as we run syncPod
-		// no matter if it changes anything, having an old version of "containers"
-		// can cause starting eunended containers.
+		// performance overhead on Docker. However, we would like to have the
```

Comment:
nit: Can we remove the first sentence and move the second sentence closer to the dockerCache call?